src/contracts/fund/policies/MaxConcentration.test.ts

```diff
@@ -1,22 +1,25 @@
-import { Eth } from 'web3-eth';
-import { HttpProvider } from 'web3-providers';
-import { Environment } from '../../../Environment';
 import { MaxConcentration } from './MaxConcentration';
+import { TestEnvironment, createTestEnvironment } from '../../../utils/tests/createTestEnvironment';
+import { MaxConcentrationBytecode } from '../../../abis/MaxConcentration.bin';
+import BigNumber from 'bignumber.js';
 
-describe('AddressList', () => {
-  let environment: Environment;
+describe('MaxConcentration', () => {
+  let environment: TestEnvironment;
   let maxConcentration: MaxConcentration;
+  let max: BigNumber;
 
-  beforeAll(() => {
-    // TODO: This should be replaced with a local ganache test environment using proper test fixtures.
-    const client = new Eth(new HttpProvider('https://mainnet.melonport.com'));
-    environment = new Environment(client);
-    maxConcentration = new MaxConcentration(environment, '0x042fcadfe10396ff5d11791357161af71ca51865');
+  beforeAll(async () => {
+    environment = await createTestEnvironment();
+
+    max = new BigNumber('100000000000000000');
```

Comment:
Why initialize that here? Just set it on line 8 in describe() direclty if it doesn't change (as a `const`)